docs/components/inputs/BytesInput.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 3
+---
+
+# TODO-BytesInput
+
+Display a `bytes` input, having a conversion tool in the input to convert from `string` to `bytes`.
+
+![BytesInput Example](/img/BytesInput.png)
+
+## Import
+
+```ts
+import { BytesInput } from "~~/components/scaffold-eth";
+```
+
+## Usage
+
+```ts
+const inputProps = {
+  name: stateObjectKey,
+  value: form?.[stateObjectKey],
+  placeholder: paramType.name
+    ? `${paramType.type} ${paramType.name}`
+    : paramType.type,
+  onChange: (value: any) => {
+    setForm((form) => ({ ...form, [stateObjectKey]: value }));
+  },
+};
+
+<BytesInput {...inputProps} />;
+```
+
+## Props
+
```

Comment:
the props are for bytes32, since this is TODO I guess you'll be changing too but maybe it could slip so just adding here

Reply:
Finally deleted this component from Docs